package.json

```diff
@@ -17,7 +17,7 @@
     "@types/aws-lambda": "^8.10.83",
     "@types/geojson": "^7946.0.8",
     "@types/node": "^20.10.4",
-    "aws-cdk": "^2.115.0",
```

Comment:
You mentioned that upgraded CDK to latest version, but the change here looks like downgrade version?

Reply:
100s to 1,000s